pandas/tests/test_algos.py

```diff
@@ -353,6 +353,25 @@ def test_on_index_object(self):
 
         tm.assert_almost_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "data, uniques, dtype_list",
+        [
+            ([1, 2, 2], [1, 2], np.sctypes["int"]),
```

Comment:
i would rather not use numpy references here and instead use our fixtures or definitions in pandas/conftest.py they are much more comprehensive and include pandas types

Reply:
Thanks for the feedback, @jreback - now using `any_numpy_dtype` to extract the dtypes.